src/EnergyPlus/WaterThermalTanks.cc

```diff
@@ -8006,30 +8041,35 @@ namespace WaterThermalTanks {
-        Real64 Eheater1 = 0.0;              // Energy change due to heater 1 over the timestep (J)
-        Real64 Eheater2 = 0.0;              // Energy change due to heater 2 over the timestep (J)
-        Real64 Eunmet = 0.0;                // Energy change unmet over the timestep (J)
-        Real64 Event = 0.0;                 // Energy change due to venting over the timestep (J)
-        int CycleOnCount1 = 0;              // Number of times heater 1 cycles on in the current time step
-        int CycleOnCount2 = 0;              // Number of times heater 2 cycles on in the current time step
-        Real64 Runtime = 0.0;               // Time that either heater is running (s)
-        Real64 Runtime1 = 0.0;              // Time that heater 1 is running (s)
-        Real64 Runtime2 = 0.0;              // Time that heater 2 is running (s)
-        bool SetPointRecovered = false;     // Flag to indicate when set point is recovered for the first time
-
-        Real64 Qheater1;            // Heating rate of burner or electric heating element 1 (W)
-        Real64 Qheater2;            // Heating rate of burner or electric heating element 2 (W)
+        Real64 Eloss = 0.0;             // Energy change due to ambient losses over the timestep (J)
+        Real64 Euse = 0.0;              // Energy change due to use side mass flow over the timestep (J)
+        Real64 Esource = 0.0;           // Energy change due to source side mass flow over the timestep (J)
+        Real64 Eheater1 = 0.0;          // Energy change due to heater 1 over the timestep (J)
+        Real64 Eheater2 = 0.0;          // Energy change due to heater 2 over the timestep (J)
+        Real64 Eunmet = 0.0;            // Energy change unmet over the timestep (J)
+        Real64 Event = 0.0;             // Energy change due to venting over the timestep (J)
+        int CycleOnCount1 = 0;          // Number of times heater 1 cycles on in the current time step
+        int CycleOnCount2 = 0;          // Number of times heater 2 cycles on in the current time step
+        Real64 Runtime = 0.0;           // Time that either heater is running (s)
+        Real64 Runtime1 = 0.0;          // Time that heater 1 is running (s)
+        Real64 Runtime2 = 0.0;          // Time that heater 2 is running (s)
+        bool SetPointRecovered = false; // Flag to indicate when set point is recovered for the first time
+        //Added three variables for desuperheater sourceinlet temperature update
+        Real64 MdotDesuperheaterWater = 0.0;      //mass flow rate of desuperheater source side water, kg/s
+        Real64 DesuperheaterPLR = 0.0;   //Desuperheater part load ratio
+        Real64 DesuperheaterHeaterRate = 0.0; //Desuperheater heater rate (W)
+        Real64 SourceInletTempSum = 0.0;    //
+        Real64 Qheater1; // Heating rate of burner or electric heating element 1 (W)
+        Real64 Qheater2; // Heating rate of burner or electric heating element 2 (W)
```

Comment:
@yzhou601 @nmerket Comment alignment is also part of clang format - so these will get pushed back anyway. All of the source files (except some of the unit test files) should be fully formatted to the specs, but for some reason VS is not interpreting the spec the same as the official clang format that was run in the initial formatting. To avoid excess formatting changes from VS2017, I've turned off the auto-format and manually apply formatting (Edit-Advanced-Format selection) to sections that I've worked on. A little tedious, but usually not a big effort. Are you using VS2017 or 2019?

![image](https://user-images.githubusercontent.com/4991075/56381300-17b4cb00-61da-11e9-85b1-4f0634116978.png)

@Myoldmopar I see there's an option to specify an alternate clang-format exe file. Maybe that would resolve the excess formatting changes.

Reply:
@mjwitte Thanks for your information, very helpful! I have switched to the manually formatting and ask VS to do nothing when I paste which is more strict lol. Do you mean that we could use the clang-format exe file to restore the format?
And I am using VS2017.

Reply:
> Do you mean that we could use the clang-format exe file to restore the format?
Possibly - I have not tried it.